Python/optimizer.c

```diff
@@ -1200,6 +1200,28 @@ int effective_trace_length(_PyUOpInstruction *buffer, int length)
 }
 #endif
 
+static bool uops_optimize_initialized = false;
+static bool uops_optimize_flag = false;
+
+static void
+initialize_uops_optimize_flag(void) {
+    if (!uops_optimize_initialized) {
+        PyInterpreterState *interp = _PyInterpreterState_GET();
+        char *env_var = Py_GETENV("PYTHON_UOPS_OPTIMIZE");
```

Comment:
Can you please do this instead at where we check for `PYTHON_JIT` ? So in `pylifecycle.c`.